src/scrcmd.c

```diff
@@ -922,7 +922,7 @@ const ScrCmdFunc Unk_020EAC58[] = {
     ScrCmd_GiveEgg,
     ScrCmd_098,
     ScrCmd_CheckPartyMonHasMove,
-    ScrCmd_09A,
+    ScrCmd_GetSlotOfPartyMonWithMove,
```

Comment:
```suggestion
    ScrCmd_FindPartySlotWithMove,
```
This is to be more in line with other commands like `ScrCmd_FindPartySlotWithSpecies`. 
Remember to change the script macros accordingly too.